### DISCUSSION TITLE
Exporting shot changes and showing the distance from the shot changes.

### DISCUSSION THREAD

**Discussion by PreetM07** [SubtitleEdit/subtitleedit#5018] [Ideas]
I wanted to check if there is any provision of exporting the generated shot changes?
Same as the import option.

Also I wanted to know if we can add a feature like showing the distance from the shot change of the in/out cue of the selected event from the nearest shot change.

This would be highly appreciated as currently there is only possibility in SE is to generate the Netflix QC report and use it. Still when you are correcting these shot change related cue errors, it doesn't help you get the exact position. As shot changes are not visible anywhere and distance is not visible.

**Comment by PreetM07** [comment]
Found this while working.
Go to C:\Users\[xxx]\AppData\Roaming\Subtitle Edit\SceneChanges

Here are the generated scene changes files. 

It has encoded names. So here is what I do to find the one I need.
1. Empty the folder.
2. Generate the shot changes from the SE.
3. There will be only one file now.
4. You can open it with notepad or ++

**Reply by OmrSi** [reply]
I think it's better yo just check the creation date of the file, which might help knowing which file you want. 

Another way it to figure out the file name from the waveforms folder, you just need to see which wave file has the same duration of the video, and you would get the file name you want.

**Reply by PreetM07** [reply]
Oh yes. Indeed a smart way. Thanks.
